spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,21 @@
+feature "Viewing a list of peeps" do
+  new_time = Time.new(2021, 10, 16, 0, 0)
+
+  around { |block| Timecop.freeze(new_time, &block) }
```

Comment:
Hadn't heard about Timecop, looks useful!

Reply:
It was super useful to stub time in the tests!